SU2_CFD/include/solver_structure.hpp

```diff
@@ -700,16 +700,23 @@ class CSolver {
    * \param[in] solver_container - Container vector with all the solutions.
    * \param[in] config - Definition of the particular problem.
    */
-  virtual void SetDissipation_Switch(CGeometry *geometry, CConfig *config);
+  virtual void SetPressure_Sensor(CGeometry *geometry, CConfig *config);
```

Comment:
I changed it to be consistent with other sensor values (e.g. the Ducros sensor), I didn't know that something else than the pressure is used in some schemes :) We can very well undo that renaming.

Reply:
Now I can remember why I did it that way. The Ducros sensor (for Roe) and the Pressure sensor (for JST) are mutually exclusive. So I wanted to avoid having an extra variable for both of them so I store them both in the `Sensor` member variable (that also saves an extra MPI routine). So we could either name them SetDissipation_Sensor/SetDucros_Sensor or SetDissipation_Switch/SetDucros_Switch.

Reply:
Ah, I see. Then, the main difference is centered vs upwind for the dissipation sensors. I am fine with either proposal, or we could even add a "centered" or "upwind" mention if you'd like.